docker-compose-mysql.yml

```diff
@@ -32,12 +40,13 @@ services:
       - MYSQL_PASSWORD=mysqlpw
 
   cdcservice:
-    image: eventuateio/eventuate-tram-cdc-mysql-service:$EVENTUATE_TRAM_VERSION
+    image: eventuateio/eventuate-tram-cdc-mysql-service:0.20.0.RELEASE
```

Comment:
Please put back $EVENTUATE_TRAM_VERSION.

Reply:
Chris, it was necessary to use local build.
I wanted to move it back, when fix is released.

Reply:
I have some not yet pushed changes that solve this problem by having two variables: EVENTUATE_TRAM_IMAGE_VERSION and $EVENTUATE_TRAM_VERSION